torch/csrc/autograd/python_function.h

```diff
@@ -34,7 +34,6 @@ struct THPFunction {
 
     PyObject *needs_input_grad;
     PyObject *backward_hooks;
```

Comment:
We expose `fn._backward_hooks` as a field. We could either reach-in and grab the PyObject* if post_hooks has a `PyFunctionPostHook` or leave this here.